### PR TITLE
Add options to change button position and icon

### PR DESCRIPTION
**Pull request by TimonVS**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,11 @@ settings] you may want:
             $('#redactor').redactor({
                 plugins: ['uploadcare'],
                 uploadcare: {
-                    // see https://uploadcare.com/documentation/widget/#configuration
+                    // styling options
+                    buttonLabel: 'Image / file'
+                    buttonBefore: 'video',
```
Comment (dmitry-mukhin) [inline README.md]: can you elaborate on this?

Comment (TimonVS) [inline README.md]: @dmitry-mukhin You mean add comments after the options?

Comment (dmitry-mukhin) [inline README.md]: can you explain me what this does (it's guessable) and why "video" is default (this is not)?

Comment (TimonVS) [inline README.md]: Ah, what this does it that you can position a button before another button. Right now the button will always be placed on the last position in the toolbar ([docs](https://imperavi.com/redactor/docs/api/button/#api-addBefore)).
Yeah, video is an example in this case. I'm not sure how I can make that clear though.

Comment (dmitry-mukhin) [inline README.md]: is default set of buttons known?

Comment (dmitry-mukhin) [inline README.md]: and what will happen if there's no "video" button?

File: README.md [hunk 2]
```diff
@@ -25,7 +25,11 @@ settings] you may want:
             $('#redactor').redactor({
                 plugins: ['uploadcare'],
                 uploadcare: {
-                    // see https://uploadcare.com/documentation/widget/#configuration
+                    // styling options
+                    buttonLabel: 'Image / file'
```
Comment (dmitry-mukhin) [inline README.md]: I would leave this to "Uploadcare" by default

**Comment by dmitry-mukhin** [comment]
Timon, I'm sorry, that was a readme, so none of my questions are really relevant.

File: uploadcare.js [hunk 3]
```diff
@@ -20,9 +20,14 @@
...
-                var button = this.button.add('uploadcare', $opts.buttonLabel || 'Uploadcare');
+                
+                var button = $opts.buttonBefore
+                    ? this.button.addBefore($opts.buttonBefore, 'uploadcare', $opts.buttonLabel || 'Uploadcare')
+                    : this.button.add('uploadcare', $opts.buttonLabel || 'Uploadcare');
+
                 this.button.addCallback(button, this.uploadcare.show);
-                this.button.setIcon(button, '<i class="fa fa-picture-o"></i>');
+
+                if ($opts.buttonIconEnabled) this.button.setIcon(button, '<i class="fa fa-picture-o"></i>');
```
Comment (dmitry-mukhin) [inline uploadcare.js]: please use {} for consistency and readability:

```js
if ($opts.buttonIconEnabled) {
  this.button.setIcon(button, '<i class="fa fa-picture-o"></i>');
}
```

**Comment by TimonVS** [comment]
The questions are still relevant actually ;). I'll test what happens if the video button doesn't exist and fix the coding style tomorrow.

**Comment by TimonVS** [comment]
Okay, the button will just get the last position if the button provided in `buttonBefore` doesn't exist. Could you merge this @dmitry-mukhin?

**Comment by dmitry-mukhin** [comment]
could you also update `history.md`

**Comment by TimonVS** [comment]
@dmitry-mukhin updated the changelog.

**Comment by dmitry-mukhin** [comment]
:(

**Comment by TimonVS** [comment]
Ah, I'll rebase

**Comment by TimonVS** [comment]
@dmitry-mukhin Fixed :)

**Comment by dmitry-mukhin** [comment]
thanks!

**Pull request merged**